thoth/package_extract/image.py

```diff
@@ -525,6 +527,42 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> Optional[str]:
+    """Get the cuda version."""
+    output = run_command("{}/usr/local/cuda/version.txt".format(path), timeout=2).stdout
+    lines = output.splitlines()
+    if len(lines) != 3:
+        _LOGGER.debug(
+            "Unable to detect CUDA version from usr/local/cuda/version.txt output: %r",
```

Comment:
You can turn all debug messages to info messages, I think this is valuable info in the log of the actual run.